.github/workflows/main.yml

```diff
@@ -14,6 +14,10 @@ jobs:
         with:
           fetch-depth: 1
 
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.18.1'
```

Comment:
OOI how did you choose this version?

Reply:
The current rush configuration appeared to want 10.x, so I declined to use the latest version at the time of writing 12.4.1. This is the currently latest available 10.x, according to running `actions/setup-node` and looking at `node -v`